cmd/options.go

```diff
@@ -4,15 +4,21 @@ import (
 	cli "github.com/jawher/mow.cli"
 )
 
+const (
+	defaultRegistryURL      = "eu.gcr.io/my-test-repository"
+	defaultReferentielImage = "dib-referentiel"
+)
+
 type buildOpts struct {
-	dryRun        bool
-	forceRebuild  bool
-	runTests      bool
-	retagLatest   bool
-	generateGraph bool
-	localOnly     bool
-	buildPath     string
-	registryURL   string
+	dryRun           bool
+	forceRebuild     bool
+	runTests         bool
+	retagLatest      bool
+	generateGraph    bool
+	localOnly        bool
+	buildPath        string
+	registryURL      string
+	referentielImage string
```

Comment:
Frenglish ?
The actual word is `referential` I think. `reference` is accurate too :
```suggestion
	referenceImage string
```